pkg/network/node/metrics.go

```diff
@@ -27,6 +27,7 @@ const (
 	PodSetupLatencyKey          = "pod_setup_latency"
 	PodTeardownErrorsKey        = "pod_teardown_errors"
 	PodTeardownLatencyKey       = "pod_teardown_latency"
+	VnidNotFoundErrorsKey       = "vnid_not_found_errors"
```

Comment:
`VNIDNotFoundErrorsKey`

Reply:
can we fix in a different patch so we don't have to restart CI?